package.json

```diff
@@ -17,9 +17,9 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@aries-framework/core": "^0.2.0-alpha.20",
-    "@aries-framework/react-hooks": "^0.1.2",
-    "@aries-framework/react-native": "^0.2.0-alpha.20",
+    "@aries-framework/core": "^0.1.0",
```

Comment:
Will this not remove revocation support?

Reply:
For now we may have to tolerate the alpha releases until 0.2.0

Reply:
Good catch. That's a mistake. I didn't realize I went the wrong way. Will fix.